pinot-perf/src/main/java/com/linkedin/pinot/perf/BenchmarkQueryEngine.java

```diff
@@ -77,34 +110,60 @@ public void startPinot() throws Exception {
       _perfBenchmarkDriver.addSegment(segmentMetadata);
     }
 
-    System.out.println("Waiting for 10s for everything to be loaded");
-    Uninterruptibles.sleepUninterruptibly(10, TimeUnit.SECONDS);
+    ZkClient client = new ZkClient("localhost:2191", 10000, 10000, new ZNRecordSerializer());
+
+    ZNRecord record = client.readData("/PinotPerfTestCluster/EXTERNALVIEW/" + TABLE_NAME);
+    while (true) {
```

Comment:
The benchmark won't start if there's something that went wrong, so the user should notice. :)